### ISSUE TITLE
muc_light_http_api_SUITE:invite_to_room/1 sometimes fails because of a deadlock in mssql

### ISSUE THREAD

**Issue by arcusfelis** [esl/MongooseIM#1912]
[test report](http://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/1911/4837/odbc_mssql_mnesia.19.3/big/ct_run.test@travis-job-c13c8885-77be-4b39-93ad-c55ea66e5254.2018-06-06_18.43.58/big_tests.tests.muc_light_http_api_SUITE.logs/run.2018-06-06_18.58.16/muc_light_http_api_suite.invite_to_room.6214.html)

```
1002 2018-06-06 18:58:21.548 [error] <0.18291.2>@mongoose_commands:
execute_command:359 Caught error:{badmatch,{error,"[FreeTDS][SQL Server]Transaction 
(Process ID 82) was deadlocked on lock resources with another process and has been 
chosen as the deadlock victim. Rerun the transaction. SQLSTATE IS: 42000"}} 
while executing invite_to_room
```

**Comment by michalwski** [comment]
That's right. I'm currently investigating it.

**Comment by michalwski** [comment]
This has been fixed by #1917
